### PR TITLE
Don't call the DBus method IsNodeFromIbft from the Storage module (#1…

### PR DESCRIPTION
**Pull request by poncovka**
…817529)

We cannot call this method from the main thread of the Storage module,
call a local function instead.

Resolves: rhbz#1817529

### CODE REVIEW

File: pyanaconda/bootloader/base.py [hunk 1]
```diff
@@ -61,16 +62,27 @@ def _get_iscsi_node_from_device(device):
...
             return False
     return True
 
 
+def _is_node_from_ibft(node):
+    """Is the node configured from iBFT table?.
+
+    :param node: the node information
+    """
+    for ibft_node in iscsi.ibft_nodes:
```
Comment (rvykydal) [inline pyanaconda/bootloader/base.py]: Looking into blivet/iscsi.py it seems that the value of ibft_nodes is not set until iscsi.setup() is called, so maybe here it is not set?

Comment (poncovka) [reply pyanaconda/bootloader/base.py]: I guess that you mean `iscsi.startup()`. We call that during the initialization of the Storage module and this code is used only by the Storage module, so ibft_nodes should be already set when we call this function.

Comment (rvykydal) [reply pyanaconda/bootloader/base.py]: Ah, it is used only by the Storage module, sorry for the noise then...

**Comment by poncovka** [comment]
@jkonecny12, the commit message was updated.

**Pull request merged**